src/test/suite/format/format.test.ts

```diff
@@ -73,6 +90,18 @@ suite('Formatter', () => {
             );
             const formatter = new FormatProvider();
 
+            // Set extension's option
+            let prevValue = null;
+            if (formatTest.extensionOption) {
+                prevValue = Global.getConfig<
+                    typeof formatTest.extensionOption.value
+                >(formatTest.extensionOption.key);
+                await Global.setConfig(
```

Comment:
We shouldn't need to call `setConfig` here with the new refactor to `options`.

Reply:
Hey @kyklish, let's remove this now :)

Reply:
Looks like we've updated this file without getting the "internal formatter" and "external formatter" tests, let's get those in before we merge this

Reply:
Resolving, not sure how I was reviewing the wrong files here 🙃